### PR TITLE
Update lambda function

### PR DESCRIPTION
**Pull request by aahm2221**

### CODE REVIEW

**Comment by oahmed-OS** [comment]
need to update the requirements.txt file with the required package `boto3` and any other packages

**Review by oahmed-OS — DISMISSED** [review]

File: lambda_function.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import json
```
Comment (oahmed-OS) [inline lambda_function.py]: can we remove this import now that we are not using it and using the dynamodb_json library instead

**Review by oahmed-OS — COMMENTED** [review]
added review

File: lambda_function.py (hunk 1 above)
Comment (aahm2221) [reply lambda_function.py]: done

**Review by oahmed-OS — APPROVED** [review]

**Pull request merged**